GliaWidgets/ViewController/Common/Alert/AlertViewController+SingleAction.swift

```diff
@@ -0,0 +1,21 @@
+extension AlertViewController {
+    func makeSingleActionAlertView(with conf: SingleActionAlertConfiguration,
+                                   confirmed: @escaping () -> Void) -> AlertView {
+        let alertView = viewFactory.makeAlertView()
+        alertView.title = conf.title
+        alertView.message = conf.message
+        alertView.actionsAxis = .horizontal
+
+        let buttonStyle = viewFactory.theme.alert.positiveAction
+
+        let button = ActionButton(with: buttonStyle)
+        button.title = conf.buttonTitle
+        button.tap = { [weak self] in
+            self?.dismiss(animated: true)
+            confirmed()
```

Comment:
imho, `confirmed` looks a little bit confused for this kind of action, since this is a one-action alert.
@kuldar-daniel, @VladKupriienko what do you think guys about this naming?